target/scripts/build/packages.sh

```diff
@@ -143,7 +144,7 @@ function _install_dovecot() {
   # when building for another architecture.
   if [[ ${DOVECOT_COMMUNITY_REPO} -eq 1 ]] && [[ "$(uname --machine)" == "x86_64" ]]; then
     _log 'trace' 'Using Dovecot community repository'
-    curl https://repo.dovecot.org/DOVECOT-REPO-GPG | gpg --import
+    curl -sSfL https://repo.dovecot.org/DOVECOT-REPO-GPG | gpg --import
```

Comment:
So a pipe operator won't cause the problem, only `&&` / `||`?

Reply:
Correct.

Pipes are no problem as we use `set -o pipefail`. This setting prevents errors in a pipeline from being masked. If any command in a pipeline fails, that return code will be used as the return code of the whole pipeline. By default, the pipeline's return code is that of the last command even if it succeeds.

Reply:
PS: Using `&&` the way it was used, makes no sense (no benefit) when `set -e` is already in place.